exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Drill2489CallsAfterCloseThrowExceptionsTest.java

```diff
@@ -477,18 +468,32 @@ public void testClosedConnectionMethodsThrowRight() {
     }
 
     @Override
+    protected boolean isOkayNonthrowingMethod(Method method) {
+      // Java 8 method
+      if ("getLargeUpdateCount".equals(method.getName())) {
+        return true; }
+      return super.isOkayNonthrowingMethod(method);
+    }
+
+    @Override
     protected boolean isOkaySpecialCaseException(Method method, Throwable cause) {
       final boolean result;
       if (super.isOkaySpecialCaseException(method, cause)) {
         result = true;
       }
+      else if (   method.getName().equals("executeLargeBatch")
+               || method.getName().equals("executeLargeUpdate")) {
+        // New Java 8 methods not implemented in Avatica.
```

Comment:
Please open a ticket for this, so we make changes once Avatica supports Java 8, add TODOs instead of comments for easy tracking ([example](https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java#L103)).